docs/concepts/trading-framework/market-protections.md

```diff
@@ -0,0 +1,164 @@
+
+Now:
+  * If an incoming order would get matched so that the price of any of the resulting trades is less than `90` or more than `110`, then that order won't be processed in the default trading mode, the market will go into a price monitoring auction, the order will get processed in the auction mode (if order type is valid for an auction), and after 6 minutes the relevant (if any) trades will be generated as per the order book state at that time. The market will return to its default trading mode and price monitoring bounds will be reset, with price ranges depending on the last mark price available.
+  * If an incoming order would get matched so that the price in any of the resulting trades is in the `[90,95]` or `[105,110]` range, the market goes into a price monitoring auction with the initial duration of 1 minute.
+    * If after 1 minute has passed there are no trades resulting from the auction or the indicative price of the auction, then if in the `[95,105]` the trades are generated and the price monitoring auction concludes.
+    * If after 1 minute has passed the indicative price of the auction is outside the `[95,105]`, the auction gets extended by 5 minutes, as concluding the auction at the 1 minute mark would breach the valid ranges implied by the second trigger. After the 5 minutes, trades (if any) are generated irrespective of their price, as there are no more active triggers, and the price monitoring auction concludes.
+
+## Liquidity monitoring
+Besides the obvious appeal to traders, a liquid market also offers some risk management, particularly in a system that does not have a central counter-party. When a trader is distressed, their position can only be liquidated if there is enough volume on the order book to offload it.
+
+In order to ensure there is enough liquidity to keep a market active and protect against insolvent parties, the network must be able to detect when the market's liquidity is too low. 
+
+When a market's liquidity drops below the safe level, the market enters into a liquidity monitoring auction, and terminates the auction when the market liquidity level is back at a sufficiently high level. The liquidity mechanics of the Vega protocol mean there is an incentive (through fee-setting) to provide the necessary liquidity.
+
+If a market enters into a liquidity auction and never again attracts enough liquidity to exit it, the market will stay in a liquidity auction until the market's settlement. Once the market's settlement price is emitted by the data source, then all market participants are settled based on their positions and account balances.
+
+There are two scenarios that can trigger liquidity monitoring mechanisms: 
+
+**If a market has too little liquidity**: Vega continuously calculates whether each market has a sufficient amount committed. When markets are deemed insufficiently liquid, they are placed into auction.
+
```

Comment:
The way that LPs commit liquidity means that as long as there are static bid and ask orders on the book the system deploys enough volume (at the specified offsets that are within price monitoring bounds and have at least the minimum probability of trading set by a network parameter). 

As a consequence a market may be deemed illiquid in two cases: 1) the total supplied stake by all LPs is below the target stake (which is a multiple of the maximum open interest over a period of time set by a network parameter). 2) the best static bid or best static ask are missing from the book, so that LP volume implied by their commitment cannot be deployed.

Reply:
As I work to incorporate this, I now think that the content about an LP not being able to cover their commitment doesn't actually fit here, and I think that also means the section on "Liquidity obligation calculations" doesn't either. What do you think @davidsiska-vega?